user_templates/mcu_defaults/ercf/Mellow_fly_ERCF.cfg

```diff
@@ -35,4 +35,3 @@ aliases:
     EXTRA_PINS9=MCU_P28  , EXTRA_PINS10=MCU_P12 ,
     EXTRA_PINS11=MCU_P29 , EXTRA_PINS12=MCU_P24 ,
     EXTRA_PINS13=MCU_P25 , EXTRA_PINS14=MCU_P13 ,
-
```

Comment:
Can you put back this second free new line at the end? It was there on purpose to allow better separation when injecting the templates from the install script